types/types.ts

```diff
@@ -150,6 +150,7 @@ export interface TermInfo {
   termId: string;
   subCollege: string;
   text: string;
+  maxEndDate: number;
```

Comment:
This should reflect the actual type in Prisma
```suggestion
  maxEndDate?: number;
```